### ISSUE TITLE
Disambiguate models and genotypes in app views

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/monarch-legacy#1165]
This is likely not an app issue but will place this here.

If I go to:
http://monarchinitiative.org/disease/DOID:9258

I can see several models contain the Pax3 gene.  However if I go to the gene page

http://monarchinitiative.org/gene/MGI:97487, only one model shows up.

My guess is that this is an issue with defining patterns for genotypes vs models, so we are capturing these models as genotypes on the gene page, but this confusing from a ux perspective.

**Comment by kshefchek** [comment]
One idea is that we have a blank node for models when a genotype is a model of disease, and show a genotype on the models tab along with this blank node label.  cc @mbrush.  Will require some refactoring in dipper as well.

**Comment by jmcmurry** [comment]
@mbrush any thoughts here?

**Comment by mbrush** [comment]
Yes. several thoughts.  We discussed the many issues related to how disease models are handled  in the data and displayed in the app with Nicole W at the Feb hackathon before she left, and settled on the approach Kent alluded to above.   Will take some work, but we will try to find time soon to address this.  Noting some more specific ideas/proposals for implementing below.
- We first need to identify all sources that provide model_of assertions, and identify what entity these are asserted on
- In our dipper models, create a distinct node for the disease model that represents a model organism strain, that is separate from the genotype, and typed as an instance of an NCBITaxon class. In the data, assert the is_model_for relation on this node.
- Create labels indicating these nodes represent organisms/models, not genotypes, e.g.:
  - for MGI, ZFIN (model asserted on the genotype): [genotype label] + 'mouse/fish'
  - for flies (model asserted on the allele): 'fly with' + [allele] + 'variant'
  - for worms (model asserted on the gene):  'worm with variant in' + [gene name] + 'gene'
- Link these NCBITaxon 'model' nodes to their respective genotype or allele nodes (using has_genotype relation)
- Update cypher queries to index and display all useful info in the app
- On **Model** pages, show:
  - Phenotypes linked to the entity asserted as a model (e.g. the genotype or allele or gene)
  - Diseases it is a model for
  - Genotype of the model (for mouse, fish, flies)
  - Genes altered in the model
  - Variants present in the model
- On **Genotype** pages, we should populate the models tab with models that contain this genotype (and show the diseases they are asserted as model for)
- On **Variant** pages - should we populate the Models tab with  models containing the variant in their genotype?
- On **Gene** pages, should wee populate the Models tab with models where the gene is altered?
- The tool tips when you hover over tabs need to be more informative - can we make different template for each combination of page and tab type? (e.g. Models tab on a Gene page should say "Disease or phenotype models that contain alterations in this gene", and Diseases tab on a Gene page should say "Diseases associated with this gene"
- Other things I noted needed fixing on the Disease page linked [above](http://monarchinitiative.org/disease/DOID:9258):
  - get species to appear for OMIA breeds
  - get species to appear for worm models
  - get gene to appear for Pax3 genotype [here](https://monarchinitiative.org/genotype/MGI:5523972) (and figure out why not appearing so can fix broadly)
